library/src/main/java/com/novoda/downloadmanager/domain/DownloadStatus.java

```diff
@@ -11,7 +11,6 @@
     public static DownloadStatus from(DownloadStage stage) {
         switch (stage) {
             case QUEUED:
-            case SUBMITTED:
```

Comment:
a `DownloadStatus` should *always*  link to a Download stage, otherwise downloads will disappear/flicker/become statusless